---
layout: archive
title: "Software"
permalink: /software/
author_profile: true
redirect_from:
  - /resume
---

***

# Genetic Risk


**TADA**: Testing for genetic association to identify risk genes using de novo and inherited genetic variants. The newest version is described  [here](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9653013/), with code on [Github](https://github.com/talkowski-lab/TADA_2022).

**DAWN**: A framework to identify risk genes and subnetworks using gene expression and genetics. Revised code to be posted soon. For current code, send email request.

**UNICORN**: A data harmonization pipeline to leverage external controls and boost power in GWAS. [Code](https://github.com/mikkoch/unicorn-qc).

***
# Methods for modeling bulk and single-cell RNA-seq data


**sLED**: Testing differences between high-dimensional covariance matrices. [Code](https://github.com/lingxuez/sLED).


**MIND**: Using multiple measurements of tissue to estimate subject- and cell-type-specific gene expression. [Code](https://github.com/randel/MIND).

 
**bMIND**: Bayesian estimation of cell type-specific gene expression with prior derived from single-cell data. [Code](https://github.com/randel/MIND).


**MarkerPen**: Identification of cell-type-specific marker genes from co-expression patterns in tissue samples. [Code](https://CRAN.R-project.org/package=markerpen).


**PisCES**: Global spectral clustering in dynamic networks. [Code](https://github.com/letitiaLiu/PisCES).


**URSM**: A Unified RNA Sequencing Model (URSM) for joint analysis of single cell and bulk RNA-seq data. [Code](https://github.com/lingxuez/URSM).

***
# Methods for modeling single-cell data


**SOUP**: Semi-soft clustering for single-cells in developmental trajectoris. [Code](https://github.com/lingxuez/SOUPR).


**locCSN**: Constructing local cell-specific networks from single-cell data. [Code](https://github.com/xuranw/locCSN).


**aLDG**: Moving from local to global gene co-expression estimation using single-cell RNA-seq with the aLDG dependence measure and others. [Code](https://github.com/JINJINT/aLDG).


**SCEPTRE**: Analysis of single-cell CRISPR screen perturbation experiments. [Code](https://github.com/Katsevich-Lab/sceptre).


**GLMeiv**: Exponential family measurement error models
for single-cell CRISPR screens. [Code](https://github.com/timothy-barry/glmeiv).


**scVAEIT**: Probabilistic modeling for single-cell multimodal mosaic integration and imputation. [Code](https://github.com/jaydu1/scVAEIT).


